tests/e2e/vsan_test.go

```diff
@@ -63,6 +64,10 @@ func (s *VsanTestSuite) SetUpTest(c *C) {
 }
 
 func (s *VsanTestSuite) TearDownTest(c *C) {
+	if len(s.containerList) > 0 {
+		out, err := dockercli.RemoveAllContainers(s.config.DockerHosts[0])
```

Comment:
This is scary and stop us moving to run test in parallel. It is better to remove the container created by this test.